### PR TITLE
Next version prepwork

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

File: e2e-tests/upgrade-consistency/run [hunk 1]
```diff
@@ -53,6 +53,16 @@ main() {
...
     compare_kubectl statefulset/${CLUSTER}-rs0 "-130"
 
+    # test 1.4.0
+    API="psmdb.percona.com/v1-4-0"
+    kubectl_bin patch psmdb "${CLUSTER}" --type=merge --patch '{
+        "metadata": {"annotations":{ "kubectl.kubernetes.io/last-applied-configuration": "{\"apiVersion\":\"'"$API"'\"}" }}
+    }'
+    wait_for_running "${CLUSTER}-rs0" "1"
+
+    compare_kubectl service/${CLUSTER}-rs0 "-140"
```
Comment (delgod) [inline e2e-tests/upgrade-consistency/run]: I think we need to add "compare" files also

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| storage | failed |
| self-healing | failed |
| demand-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| self-healing | failed |
| storage | failed |
| demand-backup | failed |

File: e2e-tests/upgrade-consistency/run (hunk 1 above)
Comment (cap1984) [reply e2e-tests/upgrade-consistency/run]: done

**Pull request merged**